src/main/java/com/google/sps/data/BooksManager.java

```diff
@@ -26,22 +26,53 @@
 import java.util.ArrayList;
 import java.util.List;
 
-/**
- * Class used for managing the stock of books. 
- */
 public class BooksManager {
+  /**
```

Comment:
None of the methods in this class seem to be called anywhere? Why do they exist in this PR?